### PR TITLE
UX: chat index refactor

### PR DESCRIPTION
**Pull request by chapoi**
Before refactor:
image
image
image

After refactor:
image
image
image

If you don't see any differences between the images, that's the point, but y'know, protocol to show a before/after ;)

### CODE REVIEW

File: plugins/chat/assets/stylesheets/common/chat-channel-title.scss [hunk 1]
```diff
@@ -8,17 +8,18 @@
...
-    border-radius: 5px;
-    padding: 2px 2px 3px;
-    color: var(--primary-high);
-    height: 0.5em;
-    width: 0.5em;
-    left: calc(0.6125em + 3px);
-    top: -4px;
-  }
+  //legacy?
+  // .category-chat-private .d-icon {
```
Comment (jjaffeux) [inline plugins/chat/assets/stylesheets/common/chat-channel-title.scss:12]: You can remove this and others in other file yes

File: plugins/chat/assets/stylesheets/common/chat-drawer.scss [hunk 2]
```diff
@@ -146,9 +134,10 @@ body.composer-open .chat-drawer-outlet-container {
         height: 1.25em;
       }
     }
-    .category-chat-private .d-icon {
-      background-color: var(--primary-very-low);
-    }
+    //legacy?
+    // .category-chat-private .d-icon {
```
Comment (jjaffeux) [inline plugins/chat/assets/stylesheets/common/chat-drawer.scss:138]: yep 🔥

File: plugins/chat/plugin.rb [hunk 3]
```diff
@@ -19,6 +19,8 @@
 register_asset "stylesheets/common/chat-drawer.scss"
 register_asset "stylesheets/common/chat-index.scss"
 register_asset "stylesheets/mobile/chat-index.scss", :mobile
```
Comment (jjaffeux) [inline plugins/chat/plugin.rb:21]: maybe rename these too

Comment (chapoi) [reply plugins/chat/plugin.rb:21]: common/chat-index isnt full-page necessarily, cause it applies to all things
and the other one is mobile so full-page is implied and redundant

Comment (jjaffeux) [reply plugins/chat/plugin.rb:21]: Ok

**Review by jjaffeux — APPROVED** [review]

**Pull request merged**